src/Packagist/WebBundle/Entity/PackageRepository.php

```diff
@@ -347,7 +352,8 @@ public function getDependents($name, $offset = 0, $limit = 15)
             ->where('v.development = true')
             ->andWhere('(r.packageName = :name OR dr.packageName = :name)')
             ->groupBy('p.id, p.name, p.description, p.language, p.abandoned, p.replacementPackage')
-            ->orderBy('p.name')
+            ->orderBy('p.abandoned')
```

Comment:
I suggest using ``addOrderBy`` for both, to avoid having to change 2 lines the next time we want to add something at the beginning